cpp/modules/deck.gl/core/src/lib/attribute/attribute-manager.cc

```diff
@@ -49,20 +49,21 @@ void AttributeManager::invalidateAll() {
 
 auto AttributeManager::update(const std::shared_ptr<arrow::Table>& table) -> std::shared_ptr<WebGPUTable> {
   // Create an empty output table
-  std::vector<std::shared_ptr<arrow::Field>> fields{};
-  std::vector<std::shared_ptr<WebGPUColumn>> columns{};
+  std::vector<std::shared_ptr<WebGPUField>> fields{};
```

Comment:
- Does each table need a specific type of fields and schemas? 
- Or can we reuse the arrow types?